test/suites/drift/suite_test.go

```diff
@@ -335,5 +337,67 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			}}),
 			Entry("Start-up Taints Drift", "Start-up Taint", test.ProvisionerOptions{StartupTaints: []v1.Taint{{Key: "example.com/another-taint-2", Effect: v1.TaintEffectPreferNoSchedule}}}),
 		)
+		DescribeTable("AWSNodeTemplate Drift", func(fieldName string, awsnodetemplatespec v1alpha1.AWSNodeTemplateSpec) {
+			if fieldName == "InstanceProfile" {
+				By("creating an instance profile")
```

Comment:
Should we move this over into an expectation function or separate function within this test suite since it's a bit long?